ui/shared/components/panel/sample.jsx

```diff
@@ -23,6 +23,7 @@ const Sample = React.memo(({ loadedSample, isOutdated, hoverDetails }) => (
         <Icon size="small" name="circle" color={iconColor(loadedSample, isOutdated)} />
         {loadedSample && <b>{loadedSample.sampleType}</b>}
         {loadedSample && loadedSample.datasetType === DATASET_TYPE_SV_CALLS && ' - SV'}
+        {loadedSample && loadedSample.datasetType === DATASET_TYPE_MITO_CALLS && ' - MITO'}
```

Comment:
I think, as above, that this would be neater if it checked that its not the variant data type, instead of checking for all other types. So this would be
```
loadedSample && loadedSample.datasetType !== DATASET_TYPE_VARIANT_CALLS && ` - ${loadedSample.datasetType}`

```